Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN apt-get update && \
     apt-get install -y curl
 
 RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl"
+RUN chmod +x kubectl && mv kubectl /usr/local/bin/kubectl
```

Comment:
@paologallinaharbur This was not before, and I think we need it. Otherwise, I don't understand how will he preinstallation job use the binaries.